run_tests.py

```diff
@@ -117,5 +118,11 @@ def is_included(backend):
         run_test(filename, "runtime", "lpython {infile}",
                  filename, update_reference, extra_args)
 
+    if run_with_dbg:
+        run_test(
+            filename, "run_dbg",
+            "lpython {infile} -g --debug-with-line-column --no-color",
+            filename, update_reference, extra_args)
```

Comment:
let's generate llvm `--show-llvm`, that way we can check in the reference LLVM code (with debugging symbols).

Reply:
Actually, we wan to run this, and then it fails (with a stacktrace) and we want to store that stack trace as reference results. so this is ok.